tests/testthat/test-as_noise_params.R

```diff
@@ -0,0 +1,37 @@
+test_that("`as_noise_params()` produces an error
+          when non-noise data is provided", {
+            library(magrittr)
+            expect_error(
+              object = noise_data <-
+                "https://osf.io/download//n6cp3/" %>% # pop data
+                readr::read_rds() %>%
+                as_noise_params(),
+              class = "not noise_params"
+            )
+          })
```

Comment:
Test that `as_noise_params` only process `noise_params` data. Pass `pop_data` to check it `as_noise_params` will raise an error.